Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -1010,7 +1010,7 @@ std::pair<std::string, std::string> GetBPRegInfo(u8 cmd, u32 cmddata)
 
   case BPMEM_SCISSOROFFSET:  // 0x59
   {
-    const X10Y10 xy{.hex = cmddata};
+    const S32X10Y10 xy{.hex = cmddata};
     return std::make_pair(RegName(BPMEM_EFB_TL),
```

Comment:
This is a typo and should be `RegName(BPMEM_SCISSOROFFSET)` (my mistake from #9497, but now would be a good time to fix it).